docs/integrations/microsoft-azure/azure-cache-for-redis.md

```diff
@@ -1,7 +1,8 @@
 ---
 id: azure-cache-for-redis
 title: Azure Cache for Redis
-description: Learn about the Sumo Logic collection process for the Azure Cache for Redis service.
+keywords: [azure redis; microsoft redis; azure cache redis; azure redis monitoring ]
```

Comment:
Docusaurus keywords should be separated by commas (https://docusaurus.io/docs/seo#single-page-metadata). Could you also apply this change to the other docs in this PR?

```suggestion
keywords: [azure redis, microsoft redis, azure cache redis, azure redis monitoring ]
```